src/odemis/model/_metadata.py

```diff
@@ -91,6 +91,7 @@
 MD_AT_SPECTRUM = "Spectrum"
 MD_AT_AR = "Angle-resolved"
 MD_AT_EM = "Electron microscope"
+MD_AT_FIB = "focused ion beam"
```

Comment:
Let's follow the other strings, and put an uppercase on the first word "Focused".